packages/core/test/invoke.test.ts

```diff
@@ -232,7 +235,7 @@ describe('invoke', () => {
     service.send({ type: 'FORWARD_DEC' });
   });
 
-  it('should forward events to services if autoForward: true before processing them', (done) => {
+  it('another forwarding pattern', (done) => {
```

Comment:
IMHO this test could just be removed, it was testing a specific thing which is no longer relevant and the new test title is meaningless

Reply:
👋

Reply:
Will delete